lib/release-ops.js

```diff
@@ -139,13 +155,14 @@ function parseLogs(logs) {
                 raw: match[0],
                 sha: match[1],
                 title: match[2],
-                flag: match[3] ? match[3].toLowerCase() : null,
+                flag: commitTagMap.get(match[3] ? match[3].toLowerCase() : null),
```

Comment:
Currently the map hard codes three conventional commits breaking prefixes with the `!` suffix, but it could be appended to any prefix. Is it worth doing `match[3].endsWith("!") ? "breaking" : ...` here, or are we okay manually making sure any breaking changes include one of those three prefixes during the migration?

Reply:
My preference is to force those three to be as explicit as possible.